packages/flutter/lib/src/material/chip.dart

```diff
@@ -2049,7 +2049,7 @@ class _RenderChip extends RenderBox with SlottedContainerRenderObjectMixin<_Chip
     final Color disabledColor = _disabledColor;
     final int disabledColorAlpha = disabledColor.alpha;
     if (needsCompositing) {
-      context.pushLayer(OpacityLayer(alpha: disabledColorAlpha), paintWithOverlay, offset);
+      context.pushOpacity(offset, disabledColorAlpha, paintWithOverlay);
```

Comment:
I believe you need to hold on to the layer this method returns, see https://master-api.flutter.dev/flutter/rendering/LayerHandle-class.html (and probably other invocations sides of this method).

Reply:
Thanks. This PR is not ready yet. I've split it to number of smaller ones, so that I can see what changes cause failures.  Will link your comments there:
https://github.com/flutter/flutter/pull/134708
https://github.com/flutter/flutter/pull/134768